homeassistant/components/lektrico/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "lektrico",
+  "name": "Lektrico Charging Station",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/lektrico",
+  "requirements": [
+    "lektricowifi==0.0.12"
```

Comment:
I cannot find the public repository providing this library. Could you point me to it?

Reply:
https://pypi.org/project/lektricowifi/0.0.20/

Reply:
Where is the code hosted providing this library ?

Reply:
As in: we're missing a git(hub) link to the source code.

Reply:
https://github.com/Lektrico/lektricowifi